### PR TITLE
Samples width: they need to be formatted at 85 characters or less.

### LINKED ISSUE #3478: Samples width: they need to be formatted at 85 characters or less.
Also add rule in VS settings file to throw error.

CC: @natke @shmoradims

### PR DESCRIPTION
**Pull request by codemzs**
fixes #3478

### CODE REVIEW

File: docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/DataViewEnumerable.cs [hunk 1]
```diff
@@ -17,22 +17,30 @@ public static void Example()
...
             // We can use the parameter ignoreMissingColumns to true to ignore any missing columns in the IDataView.
             // The produced enumerable will have the Latitude field set to the default for the data type, in this case 0. 
-            var rowEnumerableIgnoreMissing = mlContext.Data.CreateEnumerable<DatasetUtils.SampleTemperatureDataWithLatitude>(data,
-                reuseRowObject: true, ignoreMissingColumns: true);
+            IEnumerable<DatasetUtils.SampleTemperatureDataWithLatitude>
+                rowEnumerableIgnoreMissing =
+                    mlContext.Data
+                        .CreateEnumerable<
+                            DatasetUtils.SampleTemperatureDataWithLatitude
+                        >(data,
```
Comment (shmoradims) [inline docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/DataViewEnumerable.cs]: >> [](start = 24, length = 1)

let's not start a line with >
it looks like some logic operator

File: docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs [hunk 2]
```diff
@@ -49,21 +59,28 @@ public static void Example()
                     Console.WriteLine("");
                 }
 
-                Console.WriteLine("===================================================");
+                Console.WriteLine(
+                    "===================================================");
```
Comment (ganik) [inline docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs]: this looks ugly, just shorten the output a bit

Comment (codemzs) [reply docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs]: Either we do it for all files or we just go with the tool restricting line width to 85 characters :)

File: docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs [hunk 3]
```diff
@@ -49,21 +59,28 @@ public static void Example()
...
             };
             // Preview of the CharsUnigrams column obtained after processing the input.
             VBuffer<ReadOnlyMemory<char>> slotNames = default;
-            transformedData_onechars.Schema["CharsUnigrams"].GetSlotNames(ref slotNames);
-            var charsOneGramColumn = transformedData_onechars.GetColumn<VBuffer<float>>(transformedData_onechars.Schema["CharsUnigrams"]);
+            transformedData_onechars.Schema["CharsUnigrams"]
+                .GetSlotNames(ref slotNames);
```
Comment (ganik) [inline docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs]: ugy ... change names to CamelCase. Maybe shorten name to avoid this ugliness

Comment (codemzs) [reply docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs]: hey @ganik the only thing this change does is wraps C# code at 85 character mark. There are many things we can improve with regards aesthetics but it isn't within the scope. There are many typos too in the samples code...

File: docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs (hunk 2 above)
Comment (ganik) [reply docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs]: so why do we need to shorten to 85 chars? what are the benefits? 

---
In reply to: [278267763](https://github.com/dotnet/machinelearning/pull/3561#discussion_r278267763) [](ancestors = 278267763)

File: docs/samples/Microsoft.ML.Samples/Dynamic/SimpleDataViewImplementation.cs [hunk 4]
```diff
@@ -7,10 +7,14 @@
 namespace Samples.Dynamic
 {
     /// <summary>
-    /// The <see cref="IDataView"/> interface is the central concept of "data" in ML.NET. While many conveniences exist
-    /// to create pre-baked implementations, it is also useful to know how to create one completely from scratch. We also
-    /// take this opportunity to illustrate and motivate the basic principles of how the IDataView system is architected,
-    /// since people interested in implementing <see cref="IDataView"/> need at least some knowledge of those principles.
+    ///     The <see cref="IDataView" /> interface is the central concept of "data" in
```
Comment (ganik) [inline docs/samples/Microsoft.ML.Samples/Dynamic/SimpleDataViewImplementation.cs]: >      [](start = 7, length = 5)

not like. Leave it is as before

File: docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs (hunk 3 above)
Comment (ganik) [reply docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs]: sure, but going to public with such ugliness is not good. need to put best effort possible before deadline

---
In reply to: [278268213](https://github.com/dotnet/machinelearning/pull/3561#discussion_r278268213) [](ancestors = 278268213)

Comment (codemzs) [reply docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs]: Yes, we are doing best effort before the deadline. @shmoradims had specifically asked to only fix the line width. Lets see what he feels about this ugliness...

File: docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs (hunk 2 above)
Comment (codemzs) [reply docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs]: @ganik Its a requirement from the docs team. @natke can explain more.

Comment (natke) [reply docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs]: Don't blame the messenger! If you includes samples that are less than 85 chars, then readers do not have to scroll left and right to read them.

File: docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs [hunk 5]
```diff
@@ -27,20 +28,29 @@ public static void Example()
...
-            var transformedData_twochars = twoCharsPipeline.Fit(trainData).Transform(trainData);
+            var transformedData_onechars =
+                oneCharsPipeline.Fit(trainData).Transform(trainData);
+            var transformedData_twochars =
+                twoCharsPipeline.Fit(trainData).Transform(trainData);
 
             // Small helper to print the text inside the columns, in the console. 
-            Action<string, IEnumerable<VBuffer<float>>, VBuffer<ReadOnlyMemory<char>>> printHelper = (columnName, column, names) =>
+            Action<string, IEnumerable<VBuffer<float>>, VBuffer<ReadOnlyMemory<char>>
+            > printHelper = (columnName, column, names) =>
```
Comment (shmoradims) [inline docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs]: >            > [](start = 0, length = 13)

let manually fix any line that starts with >

this looks like lambda with errors.

File: docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs (hunk 2 above)
Comment (shmoradims) [inline docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs]: >=================================================== [](start = 21, length = 51)

let's also manually fix this to fit in one line.

**Comment by shmoradims** [comment]
Closing this for now, to not confuse Shauheen with a lingering PR.

**Pull request closed without merging**